processor/spanmetricsprocessor/processor.go

```diff
@@ -81,9 +85,9 @@ type processorImp struct {
 	latencyBounds        []float64
 	latencyExemplarsData map[metricKey][]exemplarData
 
-	// A cache of dimension key-value maps keyed by a unique identifier formed by a concatenation of its values:
+	// An LRU cache of dimension key-value maps keyed by a unique identifier formed by a concatenation of its values:
 	// e.g. { "foo/barOK": { "serviceName": "foo", "operation": "/bar", "status_code": "OK" }}
-	metricKeyToDimensions map[metricKey]pdata.AttributeMap
+	metricKeyToDimensions *cache.Cache
```

Comment:
Storing this as global within the package could be problematic and may cause issues in future.

Looking through the UI, it is hard to tell where this is being used or configured.

Reply:
sorry, what do you mean by "as global"?  I think the `newProcessor` function will create this variable as a new instance every time.

Reply:
Apologies, my view in github had it as though it was part of the package variables.